l1-contracts/src/core/RollupCore.sol

```diff
@@ -185,7 +184,6 @@ contract RollupCore is
     override(IStakingCore)
     returns (bool)
   {
-    setupEpoch();
```

Comment:
This can be removed as the snapshot enforces that changes will not happen to the set until the next epoch